### ISSUE TITLE
Not Training on custom dataset

### ISSUE THREAD

**Issue by sauravns** [AliaksandrSiarohin/first-order-model#465]
I was trying to train on a custom dataset, it is not showing any progress even after training for 12hrs. 
I have used the same dataset for monkey net, and it worked fine, so couldn't figure out what the issue is. It just shows a few warnings:

"/test/first-order-model/env/lib/python3.7/site-packages/torch/nn/functional.py:2423: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
/test/first-order-model/env/lib/python3.7/site-packages/torch/nn/functional.py:1332: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
/test/first-order-model/env/lib/python3.7/site-packages/torch/nn/parallel/_functions.py:61: UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.
  warnings.warn('Was asked to gather along dimension 0, but all '

and the training process is not progressing as the log file was empty. Can someone help me out.

**Comment by AliaksandrSiarohin** [comment]
Set num_repeats to 1.

**Comment by sauravns** [comment]
yea Its working now. Thanks.
what does num_repeats represent?

**Comment by AliaksandrSiarohin** [comment]
Some dataset have short epochs only few steps, to make epochs longer this parameter is used.
